problems/8.types.js

```diff
@@ -17,8 +17,12 @@
  * @param {number} b - the second number
  * @returns {boolean} - if a and b are both number types, return true
  */
-function bothNumbers() {
+// function bothNumbers(num1, num2) {
+//   return typeof(num1) === typeof(num2)
+// }
 
+const bothNumbers = (num1, num2) => {
+  return typeof(num1) === typeof(num2)
 }
```

Comment:
This will only check to see if both num1 and num2 have the same type. How might you change it to see if both are numbers.